src/components/PlsIDForm.tsx

```diff
@@ -0,0 +1,82 @@
+import * as React from "react";
+import { Form } from "library-simplified-reusable-components";
+import { connect } from "react-redux";
+import ActionCreator from "../actions";
+import { FetchErrorData } from "opds-web-client/lib/interfaces";
+import { Store } from "redux";
+import { State } from "../reducers/index";
+import { LibraryData } from "../interfaces";
+import { CheckSoloIcon } from "@nypl/dgx-svg-icons";
+import Input from "./reusables/Input";
+
+export interface PlsIDFormOwnProps {
+  uuid: string;
+  store: Store<State>;
+  fetchLibrary: (uuid: string) => LibraryData;
+  current?: string;
+}
+
+export interface PlsIDFormStateProps {
+  error?: FetchErrorData;
+}
+
+export interface PlsIDFormDispatchProps {
+  plsID?: (data: FormData) => Promise<void>;
+}
+
+export interface PlsIDFormProps extends PlsIDFormOwnProps, PlsIDFormStateProps, PlsIDFormDispatchProps {}
+
+export interface PlsIDState {
+  saved: boolean;
+}
+
+export class PlsIDForm extends React.Component<PlsIDFormProps, PlsIDState> {
+  constructor(props: PlsIDFormProps) {
+    super(props);
+    this.state = { saved: false };
+    this.submit = this.submit.bind(this);
+  }
+  async submit(data: FormData) {
+    await this.props.plsID(data);
+    await this.props.fetchLibrary(this.props.uuid);
+    this.setState({ saved: !this.props.error });
```

Comment:
Why are you using the error to update the state for `saved`?  I guess if there is no error, then it successfully saved? Does the server return a success or object you can use instead?

Reply:
Exactly: if there's no error, that means it saved.  The server does return something, but, given that we don't actually need to do anything with that value, I think it makes more sense to just do it like this.  We're already using this pattern in `EmailValidationForm` too.

Reply:
Yeah, but it'll be better if you use the good response for this instead of relying on the empty error. I think I had reservations about the `EmailValidationForm`, too.